internal/storage/storage_handle.go

```diff
@@ -144,6 +153,29 @@ func createHTTPClientHandle(ctx context.Context, clientConfig *storageutil.Stora
 		clientOpts = append(clientOpts, option.WithEndpoint(clientConfig.CustomEndpoint))
 	}
 
+	if clientConfig.ReadStallRetryConfig.Enable {
+		// Hidden way to modify the increase rate for dynamic delay algorithm in go-sdk.
+		// Ref: http://shortn/_417eTbNbKK
+		// Temporarily we kept an option to change the increase-rate, will be removed
+		// once we get a good default.
+		err = os.Setenv(dynamicReadReqIncreaseRateEnv, strconv.FormatFloat(clientConfig.ReadStallRetryConfig.ReqIncreaseRate, 'f', -1, 64))
+		if err != nil {
+			logger.Warnf("Error while setting the env %s: %v", dynamicReadReqIncreaseRateEnv, err)
+		}
+
+		// Hidden way to modify the initial-timeout of the dynamic delay algorithm in go-sdk.
+		// Ref: http://shortn/_xArUKvvGQZ
+		// Temporarily we kept an option to change the initial-timeout, will be removed
+		// once we get a good default.
+		err = os.Setenv(dynamicReadReqInitialTimeoutEnv, clientConfig.ReadStallRetryConfig.InitialReqTimeout.String())
```

Comment:
Same as above. Please switch setting env vars to some other way i possible.

Reply:
Not possible.